homeassistant/components/jellyfin/media_source.py

```diff
@@ -394,7 +394,13 @@ async def _build_series(
     async def _build_seasons(self, series_id: str) -> list[BrowseMediaSource]:
         """Return all seasons in the series."""
         seasons = await self._get_children(series_id, ITEM_TYPE_SEASON)
-        seasons = sorted(seasons, key=lambda k: k[ITEM_KEY_NAME])  # type: ignore[no-any-return]
+        seasons = sorted(
+            seasons,
+            key=lambda k: (
+                ITEM_KEY_INDEX_NUMBER not in k,
+                k.get(ITEM_KEY_INDEX_NUMBER, None),
+            ),
```

Comment:
Please add a comment explaining what's going on here. Also, there's no need to pass `None` to `dict.get`, it's the default.

Reply:
Done